DEVELOPER_GUIDE.md

```diff
@@ -274,33 +274,7 @@ curl localhost:9200
 
 Additionally, it is also possible to run a cluster with security enabled:
 ```shell script
-./gradlew run -Dsecurity.enabled=true -Dhttps=true -Duser=admin -Dpassword=admin
-```
-
-By default, if `-Dsecurity.enabled=true` is passed the following defaults will be used: `https=true`, `user=admin` and
-`password=admin`.
-
-Then, to access the cluster, we can run
```

Comment:
why do we need to remove example with curl command? My understanding is that password itself has been changed, but it still can be passed in the same way so example is still valid

Reply:
Ah I misread the developer guide.  I'll just change the sentence about defaults